### DISCUSSION TITLE
[Kavita] Webtoon Manual Zoom Slider

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2559] [Ideas]
### Idea Description

Posted by Avoozl
· Nov 2021
· 12 Upvotes
https://feats.kavitareader.com/posts/7/webtoon-manual-zoom-slider

Description

Love the webtoon mode, only improvement request I have is a "manual zoom slider" (so the images are not full width), sometimes I want to read at a "distance" without having my eyes have to the edges of the screen while retaining the infinite scroll mode.

Current/Normal Mode:
https://user-images.githubusercontent.com/58608430/143058529-2650086c-f2c7-42d8-b5a5-831781cd2f39.png

Example of Requested Mode using Browzer Zoom:
https://user-images.githubusercontent.com/58608430/143056475-39197b97-f173-4c5f-a221-84851019bf7e.png

Browser Zoom, Small UI:
edit: I can use the browser zoom to get the result I want, but the issue is that the UI gets smaller too, which is a bummer.
https://user-images.githubusercontent.com/58608430/143057762-a8d4da5c-c809-4835-95df-13fd9fcf4ac5.png

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by maknilsin** [comment]
Absolutely one of my most sought after features. This would make a switch from Komga a no brainer.

**Comment by thezak48** [comment]
Using this CSS override you can get images to a reasonable size while not having to zoom out
```CSS
    div.webtoon-images {
        padding-left:30%;
        padding-right:30%;
    }
 ```
 Adjust percentage to suit monitor size

**Comment by majora2007** [comment]
I believe this is the same thing: https://github.com/Kareadita/Kavita/discussions/2558

Users are looking for fit to width with some sort of padding to allow it to be more comfortable.

**Comment by majora2007** [comment]
I validated that we implemented this last year, so as of v0.8.4 you can override the width of images in webtoon or other modes.
